pages/20-01.md

```diff
@@ -478,13 +490,15 @@ These reports should be treated the same as all other reports: like a gift. Know
 Obviously, if the source is unknown, you are not under any requirement to find or reply to the person, and you should **never attempt to unmask** the identity of the person who offers a report in good faith.
 
 #### Can we recognize vulnerability reporters for their efforts?
-Yes. Some organizations publicly acknowledge people who relay confirmed or especially impactful reports on a webpage or in [social media](https://web.archive.org/web/20200730203649/https://twitter.com/DC3VDP/status/1278664041528000512?s=20). While a public 'thank you' is a generous practice, it must not be done unless you've received explicit permission from the reporter that they are comfortable with public acknowledgement.
+Yes. Some organizations publicly acknowledge people who relay confirmed or especially impactful reports on a webpage or in [social media](https://web.archive.org/web/20200730203649/https://twitter.com/DC3VDP/status/1278664041528000512?s=20). A public 'thank you' is a generous practice, costs nothing, and make vulnerability reporters happy to share their findings in the future. However, it must not be done unless you've obtained explicit permission from the reporter that they are comfortable with public acknowledgement.
```

Comment:
A public 'thank you' is a generous practice, costs nothing, and make vulnerability reporters happy to share their findings in the future. 

make should be makes